pex/pex_warnings.py

```diff
@@ -6,13 +6,20 @@
 
 import warnings
 
+from pex.typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from pex.pex_info import PexInfo
+    from pex.variables import Variables
+
 
 class PEXWarning(Warning):
     """Indicates a warning from PEX about suspect buildtime or runtime configuration."""
 
 
 def configure_warnings(pex_info, env):
-    if env.PEX_VERBOSE > 0:
+    # type: (PexInfo, Variables) -> None
+    if env.PEX_VERBOSE is not None and env.PEX_VERBOSE > 0:
```

Comment:
This was a bug, although I don't know if we ever hit it. It's possible for `PEX_VERBOSE` to be `None`, such as if you call `ENV.strip()` and the user did not define the value.